XenAdmin/ConsoleView/RdpClient.cs

```diff
@@ -55,11 +55,8 @@ class RdpClient: IRemoteConsole
         /// </summary>
         private MsRdpClient9 rdpClient9 = null;
 
-        private MsRdpClient6 rdpClient6 = null;
-
         /// <summary>
-        /// This will be equal to rdpClient9, if the DLL that we've got is version 8, otherwise equal to
-        /// rdpClient6.
+        /// This will be equal to rdpClient9

```

Comment:
I dont't think we need both rdpControl and rdpClient9 members, because now they are always equal.